crates/sui-graphql-rpc/src/types/query.rs

```diff
@@ -214,17 +217,52 @@ impl Query {
         version: Option<UInt53>,
     ) -> Result<Option<Object>> {
         let Watermark { checkpoint, .. } = *ctx.data()?;
+        let key = match version {
+            Some(version) => Object::at_version(version.into(), checkpoint),
+            None => Object::latest_at(checkpoint),
+        };
 
-        match version {
-            Some(version) => {
-                Object::query(ctx, address, Object::at_version(version.into(), checkpoint))
-                    .await
-                    .extend()
-            }
-            None => Object::query(ctx, address, Object::latest_at(checkpoint))
-                .await
-                .extend(),
-        }
+        Object::query(ctx, address, key).await.extend()
+    }
+
+    /// The package corresponding to the given address at the (optionally) given version.
```

Comment:
```suggestion
    /// The package corresponding to the given address (at the optionally given version).
```
I might be reading this wrong, but I think the whole clause is parenthetical.